anet.yml

```diff
@@ -152,8 +152,12 @@ dictionary:
   ADMINISTRATOR_POSITION_TYPE_TITLE: ANET Administrator
   PRINCIPAL_ORG_NAME: Afghan Government Organization
   ADVISOR_ORG_NAME: Advisor Organization
-  POAM_LONG_NAME: Plan of Action and Milestones / Pillars
-  POAM_SHORT_NAME: PoAM
+  TASK_LONG_NAME: Plan of Action and Milestones / Pillars
+  TASK_SHORT_NAME: Task
```

Comment:
I think wou also need the plural for this one, as we sometimes use Tasks (see insights, where we use now All tasks).

Reply:
Maybe for now we can get away by doing a + "s" when plural is needed

Reply:
For now I agree with Vasil, maybe at some point a pluralize method needs to be added...

Reply:
@maradragan - to lower the threshold for rename detection, in this case you can do something like `git diff -M40% candidate..GH-323`. Unfortunately, I don't know how to tell github to do this, so command line only I guess